Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -242,172 +254,220 @@ edgebridgetutorialapp://
 
-5. In the JSON property viewer window, click the dropdown arrows next to `data` (**1**) and `contextdata` (**2**).
-6. Then select the first property to map, `product.add.event` (**3**) and click `Select` (**4**).
+<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-mapping-json.png" alt="Datastream mapping" width="1100"/>  
 
-<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-mapping-json-property.png" alt="Datastream select property" width="1100"/>  
+3. In the JSON property selector window, select the dropdown chevrons next to **data** -> **__adobe** -> **analytics** -> **contextData** (**1**).
+4. Then choose the property to map, **myapp.category** (**2**), and select **Select** (**3**).
 
-Notice that in the property viewer, you can see the data hierarchy, where `data` is at the top, `contextdata` is one level down, and `product.add.event` is one level below that. This is nested data, which is a way to organize data in the JSON format.
+<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-mapping-json-property.png" alt="Datastream select property" width="750"/>  
 
-> **Info**
-> The data mapper interprets the `.` character as nesting, which means if there are `.` characters in a property name that are not meant to be nesting, namely the ones in our current example: `product.add.event`, we need to escape this behavior by adding backslashes `\` before the `.` (**1**).
+> [!IMPORTANT]
+> When interacting with hierarchies, if a child attribute contains a period (`.`), use a backslash (`\`) to escape special characters. For more details, refer to the guide on [escaping special characters](https://experienceleague.adobe.com/en/docs/experience-platform/data-prep/home#escape-special-characters).
 
-7. Add backslashes `\` before the `.` characters as shown below (**1**).
+5. Add a backslash `\` before the `.` character in `myapp.category` as shown below (**1**).
+    - Example: `myapp.category` -> `myapp\.category`
 
-Now, we need to map this JSON property from the Edge Bridge event to its matching property in the XDM schema.
+Now that the source field setup is complete, the next step is to map this property to its corresponding target property in the XDM schema.
 
-8. Click the schema icon (**2**) to open the XDM property viewer window.
+6. Select the schema icon (**2**) to open the XDM property viewer window.
 
 <img src="../assets/edge-bridge-tutorial/data-prep/datastreams-mapping-xdm.png" alt="Datastream mapping XDM" width="1100"/>  
 
-9. In the XDM property viewer window, click the dropdown arrows next to `commerce` (**1**) and `productListAdds` (**2**).
-10. Then select the `value` property (**3**) and click `Select` (**4**).
+7. In the XDM property viewer window, select the dropdown chevron next to **application** (**1**).
+8. Then choose the **closeType** property (**2**) and select **Select** (**3**).
```

Comment:
any strong reason why we are mapping the myapp.category to application.closetype? this is a field already populated  by the Lifecycle extension, so might be best to use another standard / custom field for the app category.

Reply:
custom field sounds great